tests/testBackground.py

```diff
@@ -513,22 +458,8 @@ def testOnlyOneGridCell(self):
                 testval = afwMath.cast_BackgroundMI(backobj).getPixel(bctrl.getInterpStyle(), xpix, ypix)
                 self.assertAlmostEqual(testval/mean, 1)
 
-    def testTicket1681OffByOne(self):
```

Comment:
No, it would be reported as an "Error", not a test "Failure". Those are explicitly different things, and it's easier to debug if we're explicit about Failing tests so that real errors (e.g. someone using the wrong call signature) are clearly distinct from failed tests.

If you really do want to allow tests without asserts or explicit failure conditions (e.g. ones that "Error" when something goes wrong), then they need to be clearly documented as such, not just a bunch of random function calls.